src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -150,6 +150,19 @@ public async Task<IEnumerable<DependencyDetail>> ParseVersionDetailsXmlAsync(str
                 }
 
                 XmlNode nodeToUpdate = versionList.Item(0);
+
+                // If the 'Pinned' attribute does not exist or if it is set to false we just not update it 
```

Comment:
Checking for the existence of the "Pinned" attribute is for backwards compat purposes. We could also add `Pinned="false"` if is not defined yet but as is assumes not defined == false